docs/hw/face.md

```diff
@@ -63,10 +63,10 @@ The LEDs expose internal state information about the robot, but can also be over
 |  Device is connected to <br> robot’s Access Point page.  |  Robot downloading <br> update file  |  Robot updating firmware <br> Do not remove from dock  |  Update successful  |
 
 ### While Operating
```

Comment:
Looks like there might be a spacing issue with the table that is squishing three of the gifs. Looks like it does that on the current page as well, but only to one of them. At first glance, I cannot identify any issues in the code. It may be this section needs a second table like in the "While Connecting to Wi-Fi" section
<img width="773" alt="Screenshot 2024-01-09 at 6 47 06 PM" src="https://github.com/iRobotEducation/create3_docs/assets/66318600/3af23836-26f2-412c-8de8-cbc7f17e14d4">

Reply:
Blurg.

Reply:
This squishing is thanks to markdown tables wanting to be smart, I think. I am not enough of a CSS expert to fix it.

Reply:
One thing I can do is to increase the maximum width of the site, so that it looks good if you have a wide monitor. I could also make the images smaller. I will try a few other things before giving up. ;-)

Reply:
(cc @Rebecca-iRobot )

Reply:
Good news, this totally fixed it on Chrome! Looks great there...bad news, on Safari it is worse 
<img width="801" alt="Screenshot 2024-01-10 at 9 32 08 PM" src="https://github.com/iRobotEducation/create3_docs/assets/66318600/344acaed-248c-4a23-bd46-08d4816a8a36">

But I can sacrifice since I'm probably the only person who uses Safari :)

Reply:
I just checked Firefox and it looks great there, too. Apparently Safari has like ~20% share on the browser market, so we shouldn't be flip about it. The only other idea I have is to make a table of tables; maybe if I do that, instead of squooshing the cells, it might make line breaks happen?

Reply:
The other other idea I have is to make the whole silly thing a preformatted image, but that will be a real pain to maintain.